central/complianceoperator/v2/report/manager/watcher/scanwatcher.go

```diff
@@ -0,0 +1,239 @@
+package watcher
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/pkg/errors"
+	complianceIntegrationDS "github.com/stackrox/rox/central/complianceoperator/v2/integration/datastore"
+	scanDS "github.com/stackrox/rox/central/complianceoperator/v2/scans/datastore"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/concurrency"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/protocompat"
+	"github.com/stackrox/rox/pkg/search"
+	"github.com/stackrox/rox/pkg/set"
+	"golang.org/x/mod/semver"
+)
+
+var (
+	log                                                = logging.LoggerForModule()
+	ScanTimeoutError                                   = errors.New("scan watcher timed out")
+	ComplianceOperatorNotInstalledError                = errors.New("compliance operator is not installed")
+	ComplianceOperatorVersionError                     = errors.New("compliance operator version")
+	ComplianceOperatorIntegrationDataStoreError        = errors.New("unable to retrieve compliance operator integration")
+	ComplianceOperatorIntegrationZeroIntegrationsError = errors.New("no compliance operator integrations retrieved")
```

Comment:
We have a convention that vars for errors start with `Err`, so this could be written as:
```suggestion
	ErrScanTimeout                                   = errors.New("scan watcher timed out")
	ErrComplianceOperatorNotInstalled                = errors.New("compliance operator is not installed")
	ErrComplianceOperatorVersion                     = errors.New("compliance operator version")
	ErrComplianceOperatorIntegrationDataStore        = errors.New("unable to retrieve compliance operator integration")
	ErrComplianceOperatorIntegrationZeroIntegrations = errors.New("no compliance operator integrations retrieved")
```